scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3621,6 +3630,22 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     val eparam = inParens(
       param(ownerIsCase = false, ownerIsType = false, isImplicit = false, isUsing = false)
     )
+    val uparams = if (token.is[LeftParen]) {
```

Comment:
We can have multiple using blocks:
```
Extension         ::=  ‘extension’ [DefTypeParamClause] ‘(’ DefParam ‘)’
                       {UsingParamClause} ExtMethods
```
`{}` means it can get repeated.

This compiles:
```
    extension (i: Int)(using u : String)(using t : java.io.File) 
```

Reply:
fixed :+1: